tests/Support/InMemoryAccountRepository.php

```diff
@@ -33,6 +34,21 @@ public function get(AccountId $id): Account
         return $this->accounts[$id->value] ?? throw new AccountNotFoundException();
     }
 
+    public function find(AccountFilter $filter): array
+    {
+        return array_values(
+            array_filter(
+                $this->accounts,
+                static function (Account $account) use ($filter): bool {
+                    if ($filter->getAccountChartId() !== null && !$account->chartId->isEqualTo($filter->getAccountChartId())) {
+                        return false;
+                    }
+                    return true;
```

Comment:
```suggestion
                    return $filter->getAccountChartId() === null || $account->chartId->isEqualTo($filter->getAccountChartId());
```

Reply:
Also, should it return `true` on `null` chart ID?

Reply:
> Also, should it return `true` on `null` chart ID?

yes